clock/README.md

```diff
@@ -5,6 +5,362 @@ A Go time testing library for writing deterministic unit tests
 _Note: Quartz is the name I'm targeting for the standalone open source project when we spin this
 out._
```

Comment:
Do you know when this would be? Is it worth calling this `quartz` here instead now?

Reply:
I think it will be soon.  We could rename to `quartz` in-repo before spinning out, or at the time we spin out. I don't think one is particularly easier or harder.